wordpress-mobile/WordPress-iOS
# History / Revisions: display author informations

Refs. #10304 

This small PR displays the author informations, within a Revision cell, we got after a blog sync operation.
A `BlogAuthor` will provide the _username_ and the _avatar image_ as displayed data.

![revisions-list-authors](https://user-images.githubusercontent.com/912252/48549633-3976f280-e8c8-11e8-9339-c13bb2cdd0c3.jpg)

**To test:**
- Open a post with some revisions
- If the author exists then you should see the avatar image and the username.
- If the author doesn't exist then you should see the avatar image placeholder and a _time ago_ date
